shopfloor/models/stock_move.py

```diff
@@ -104,7 +104,6 @@ def _extract_in_split_order(self, default=None, backorder=False):
         self.package_level_id.picking_id = new_picking.id
         self.move_line_ids.picking_id = new_picking.id
         self.move_line_ids.package_level_id.picking_id = new_picking.id
-        self._action_assign()
```

Comment:
Why not doing:
```python
if self.state != "assigned":
    self._action_assign()
```
(I'm sure there is a reason, but it was the fix done elsewhere IIRC)